main.go

```diff
@@ -1,4 +1,4 @@
-package main // import "4d63.com/gochecknoglobals"
+package main
```

Comment:
This change is unrelated to the PR, can you revert it? Let's discuss issues with this in a new issue if this is causing problems.

```suggestion
package main // import "4d63.com/gochecknoglobals"
```